Automattic/Simplecons
# Optimise SVG Code - Remove Multiple Paths

This is the second PR to optimise the SVG code of Simplecons.

The goal of optimisation as a whole is to:
- Make sure all icons only contain a single path.
- Make sure all icons have their path decimal precision set to 3 decimal places.

In this PR I am modifying icons that only contain multiple `path` tags to reduce them to a single path.

The icons affected by this change are:

- cloud-sync
- collaborate (Icon has been shifted down by 1px for better center alignment)
- contract
- copy
- expand
- note
- settings
- share
- sort-order
- visibility-hidden
- visibility-visible
- warning

### To Test
- Verify that the above list of icons now only contain one path
- Verify that the icons listed above have not changed visually compared with their non-optimsed versions.
